install_mac.sh

```diff
@@ -0,0 +1,9 @@
+# Install dependencies
+brew install python gdal spatialindex
+
+# Install pyproj
+pip3 install cython
+pip3 install git+https://github.com/jswhit/pyproj.git
```

Comment:
Installing from PyPi does not work for some reason, however installing directly from the git repository works.